packages/boxel-ui/addon/components/add-button/index.gts

```diff
@@ -1,12 +1,14 @@
 import type { TemplateOnlyComponent } from '@ember/component/template-only';
 import IconButton from '../icon-button';
-import { eq } from '../../helpers/truth-helpers';
+import { eq, bool } from '../../helpers/truth-helpers';
 import { svgJar } from '../../helpers/svg-jar';
 
 interface Signature {
   Element: HTMLElement;
   Args: {
-    variant?: 'full-width';
+    variant?: 'full-width' | 'pills';
```

Comment:
I think this should be `pill` instead of `pills`. It is one button, which looks like a pill when this variant is set

Reply:
Updated